libsplinter/src/error/internal.rs

```diff
@@ -0,0 +1,281 @@
+// Copyright 2018-2020 Cargill Incorporated
```

Comment:
looks like you are missing the mod.rs file

Reply:
doh, added